src/myprofile-page/CreateMutationHelper.js

```diff
@@ -0,0 +1,115 @@
+import { gql } from '@apollo/client';
+import Moment from 'moment';
+import { PLACEHOLDER_MUTATION } from '../sources';
+
+export const createUserEducations = (userId, educations) => {
+  if (!(educations && educations.length > 0)) {
+    return PLACEHOLDER_MUTATION;
+  }
+  const mutations = [];
+  for (let index = 0; index < educations.length; index += 1) {
+    const {
+      institution, description, startDate, endDate,
+    } = educations[index];
+    const mutation = `
+                education${index}: createUserEducation(
+                        userId: ${userId},
+                        institution: "${institution}",
+                        description: "${description}",
+                        startDate: "${Moment(new Date(startDate) * 1000).format('YYYY-MM-DD hh:mm:ss')}",
```

Comment:
You may consider keeping the format in a const and re-using it.

Reply:
fixed
`export const dateTimeFormat = 'YYYY-MM-DD hh:mm:ss';`